src/pages/QueriedEventPage/index.tsx

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+import { useLocation } from 'react-router';
+import queryString from 'query-string';
+
+import EventCreationPage from '../EventCreationPage';
+
+import { OfficerRoutingPermission } from '@HOCs/RoutingPermissions';
+
+/*
+ * This is for conditional rendering of component(s) at /events endpoint based on
+ * the endpoint's query parameters
+ */
+function QueriedEventPage(): JSX.Element {
```

Comment:
we shouldn't have a separate page for this - we should be switching on params in '/events' instead

Reply:
What do you mean switching on params in '/events'? You mean do it at the App level?? <Route/>'s path prop doesn't recognize query params

Reply:
page level, /events and /events?x=true is the same page

Reply:
Yeah but we don't have a page on /events yet, the Get Involved/Events page is on /

Reply:
wait what's the calendar under? I think it would make sense to put it as events

Reply:
It's currently under /calendar